sdk/websites/Azure.ResourceManager.AppService/tests/TestsCase/SiteInstanceProcessResourceTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.ResourceManager.AppService.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.AppService.Tests.TestsCase
+{
+    internal class SiteInstanceProcessResourceTest : AppServiceTestBase
+    {
+        public SiteInstanceProcessResourceTest(bool isAsync) : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+        [TestCase]
+        [RecordedTest]
+        public async Task SiteInstanceProcessResource_Test()
+        {
+            string resourceGroupName = "testRG";
+            string serviceName = "testapplwm";
```

Comment:
For the test resource, we better create them in the test, or add a deployment template here so that other engineers could create the resource based on that.

Reply:
tried this way but it does not have any instance. So use the existing one at present.